app/helpers/helpers.rb

```diff
@@ -0,0 +1,10 @@
+class Helpers
```

Comment:
Sinatra helpers are typically defined in a `helpers` block in the application controller. That way you won't have to pass the session -- they will already have access to it. :)